benchmarks/benchmarks.py

```diff
@@ -0,0 +1,524 @@
+""" Benchmarks for use with the airspeed velocity (asv) package. 
+(http://asv.readthedocs.io/en/latest/)
+
+The calculate and equilibrium benchmarks were chosen from the examples for 
+simplicity. They may not be completely representative, but the purpose of the
+benchmarks is to show how the performance changes over time. Testing the limits
+of the code is for the tests.
+"""
+
+from pycalphad import Database, Model, calculate, equilibrium, v
+from pycalphad.core.halton import halton
+import timeit
+
+__author__ = "Brandon Bocklund"
+
+class BenchmarkSetups:
+    """ Benchmarks the creation of databases and models.
+    
+    Note that multiple databases, files, etc. could be tested
+    using the 'params' attribute. 
+    """
+
+    def setup(self):
+        # create a file to read from
+        with open('alfe_sei.TDB', 'w') as f:
+            f.write(ALFE_TDB)
+        # setup TDBs
+        self.tdb_file =  open('alfe_sei.TDB', 'r')
+        self.tdb_string = ALFE_TDB
+        fec_tdb = FEC_TDB
+        # setup databases
```

Comment:
Per offline conversation, we do this because making a Database from a file on disk follows a different code path from a string, so this difference may be worth capturing in our benchmark.